lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -119,14 +119,15 @@ def create_milestone(repository:, title:, due_date:, days_until_submission:, day
       # @param [String] description The text to use as the release's body / description (typically the release notes)
       # @param [Array<String>] assets List of file paths to attach as assets to the release
       # @param [TrueClass|FalseClass] prerelease Indicates if this should be created as a pre-release (i.e. for alpha/beta)
+      # @param [TrueClass|FalseClass] is_draft Indicates if this should be created as a draft release (defaults to true)
       #
-      def create_release(repository:, version:, target: nil, description:, assets:, prerelease:)
+      def create_release(repository:, version:, target: nil, description:, assets:, prerelease:, is_draft: true)
```

Comment:
> I wonder if we should set a default value for is_draft in the helper here, instead of letting the responsibility of the default value to be set by the Action (via the ConfigItem's default_value: true which already provides this) that uses that helper

Another advantage of not providing a default value for `is_draft:` parameter in this helper's method is that there would then be no risk of accidentally having them out of sync with the default value provided by the `ConfigItem` of the action. Like, imagine if in the future we decide to change the `default_value` of the `ConfigItem(key: :is_draft, …)` in the `CreateReleaseAction` from `true` to `false`… but then forget to also change the default value of the `is_draft:` parameter in that helper function in concert… we'd start having inconsistencies in what default value we consider, depending on if we look at it from the context of the public API / fastlane `Action`, or from the POV of the helper, and that could become confusing. I'd say better have a single source of truth for the default value, and that source of truth being the `ConfigItem` of the `CreateReleaseAction` (it being the public entry point for this)